docs/2.8.4/docs/canton/usermanual/apis.rst

```diff
@@ -295,6 +280,44 @@ certificates of the Java runtime will be used. An example would be:
    :end-before: architecture-handbook-entry-end: TlsConnect
    :dedent:
 
+Server Authentication
+^^^^^^^^^^^^^^^^^^^^^
+
+Canton has two ways to perform server authentication to protect from man in the middle attacks. Firstly, if TLS is used on the public API as described above, TLS also takes care of server authentication as that is one of the core functions of TLS.
+
+Secondly, like all nodes, the sync domain has an identity which corresponds to the fingerprint of its namespace root key. It reports its identity to connecting participant nodes and signs all its messages with keys authorized by that namespace root key on the topology ledger. Assuming no key compromises, this gives participants a guarantee that the reported identity is authentic. Thus server authentication can also be performed by the sync domain operator passing their sync domain identity to the participant node operator, and checking that the identity matches that reported by the sync domain to the participant node. The domain identity of the sole connected sync domain can be read out using console commands like:
+
+.. code-block:: none
+  
+    participant1.domains.list_connected.last.domainId.filterString
+
+Client Authentication
+^^^^^^^^^^^^^^^^^^^^^
+
+Unlike the ledger or admin API, the public API uses Canton's cryptography and topology state for client authentication rather than mutual TLS (mTLS). To connect to the public APIs, clients need to connect in two steps:
+
+1. The client calls the `SequencerConnectService` to align on Canton Protocol versions and obtain the domain id.
+2. During the first connection, the client registers by sending its minimal topology state (identity, key delegations, public keys) to the domain manager.
+3. The client calls the `SequencerAuthenticationService` to authenticate using a challenge-response protocol and get an access token for the other sequencer services.
+4. The client connects to the main `SequencerService` using the access token from 3.
+
+The information the client provides in step 2 is verifiable since it's a certificate chain of keys. The domain rejects this if either the namespace root key fingerprint included is not permissioned  (see :ref:`permissioned-domains`), or if the topology state provided is invalid.
```

Comment:
```suggestion
The information the client provides in step 2 is verifiable since it is a certificate chain of keys. The domain rejects this if the namespace root key fingerprint included is not permissioned  (see :ref:`permissioned-domains`) or if the topology state provided is invalid.
```